refactor: `GetAuthUser` was modified.

guerretta16 opened the pull request:
#20 
- `GetAuthUser` was modified.
- Now, if user is `ADMIN` will not receive the categories.

guerretta16 committed:
refactor(controller): `GetAuthUser` was modified.

- Now, if user is `ADMIN` will not receive the categories.

f-esquivel commented on app/Http/Controllers/Auth/GetAuthUserController.php:
```diff
@@ -19,7 +19,9 @@ public function __invoke(): JsonResponse
         $user = auth()->user();
 
-        $user->load('categories');
+        if ($user->type !== UserTypesConstant::ADMIN) {
+            $user->load('categories');
+        }
```
The user model has a utility function to verify if the record is an `ADMIN`. Use that function.

guerretta16 committed:
refactor(controller): `GetAuthUser` was modified.

- isAdmin function used for that.

- Now, if user is `ADMIN` will not receive the categories.
